Library/Homebrew/cask/cmd/upgrade.rb

```diff
@@ -41,6 +42,8 @@ def run
           .map { |(old_cask, new_cask)| "#{new_cask.full_name} #{old_cask.version} -> #{new_cask.version}" }
           .join(", ")
 
+        return puts "Dry run: did not upgrade anything." if dry_run?
```

Comment:
Instead: changing the `oh1` above to be `oh1 "Would upgrade ...` and then `return` here.